src/Utils.js

```diff
@@ -213,6 +213,19 @@ export function coercePropValue(propName, propValue) {
     return propValue;
   }
 
+  // can be the empty string
```

Comment:
should these three checks here or after the trim like the ones for true and false?